temporalcli/commandsgen/commands.yml

```
@@ -2498,6 +2653,95 @@ commands:
         type: int
         description: Maximum number of Workflow Executions to display.
 
+  - name: temporal workflow modify-options
```

```suggestion
  - name: temporal workflow update-options
```

Like https://github.com/temporalio/cli/pull/729

---

This was intentional. Otherwise it gets listed under `update`, and it gets very confusing because people will assume it is a global option for updates.  Activities don't take updates, so they don't have that issue...

---

Activities may take updates in the future, but I think we should call this "update options" in both the API and the CLI. And we do for the latter. We are ok saying "update" and "update options" are different I think.

---

I also feel that it makes more sense to call this `workflow update-options`. We've discussed the modify vs update name choice considering the context of `workflow update` and `activity update-options`, and I think that `update` and `update-options` can be separate things.

---

Sure, they can be separate things, the issue is whether people will get confused with 'options of an update".
 I think using UpdateOptions in the API was a mistake, but the train may have already left the station, particularly, if activities in the CLI are already using it... Any thoughts @drewhoskins-temporal ?

---

+1 to @carlydf's take.  This should be consistent with `activity update-options` as well as the general expectation that such methods are called 'update'.

---

OK, majority rules, changing to `update-options`...